Kickstarter-iOS/AppDelegateViewModel.swift

```diff
@@ -1288,23 +1275,28 @@ extension ShortcutItem {
   }
 }
 
-private func optimizelyData(for environment: Environment) -> (String, OptimizelyLogLevelType, TimeInterval) {
-  let environmentType = environment.environmentType
-  let logLevel = environment.mainBundle.isDebug ? OptimizelyLogLevelType.debug : OptimizelyLogLevelType.error
-  let dispatchInterval: TimeInterval = 5
-
-  var sdkKey: String
-
-  switch environmentType {
-  case .production:
-    sdkKey = Secrets.OptimizelySDKKey.production
-  case .staging:
-    sdkKey = Secrets.OptimizelySDKKey.staging
-  case .development, .local, .custom:
-    sdkKey = Secrets.OptimizelySDKKey.development
-  }
-
-  return (sdkKey, logLevel, dispatchInterval)
+private func configureOptimizely(for _: Environment) -> OptimizelyClientType? {
+  // FIXME: This is until we add a new feature flagging client
+  let mockOptimizelyClient = MockOptimizelyClient()
+    |> \.experiments .~ [
+      OptimizelyExperiment.Key.nativeRiskMessaging.rawValue: OptimizelyExperiment.Variant.control.rawValue,
+      OptimizelyExperiment.Key.nativeProjectCards.rawValue: OptimizelyExperiment.Variant.control.rawValue,
+      OptimizelyExperiment.Key.onboardingCategoryPersonalizationFlow.rawValue: OptimizelyExperiment.Variant
+        .control.rawValue,
+      OptimizelyExperiment.Key.nativeOnboarding.rawValue: OptimizelyExperiment.Variant.control.rawValue
+    ]
+    |> \.features .~
+    [
+      OptimizelyFeature.commentFlaggingEnabled.rawValue: false,
+      OptimizelyFeature.consentManagementDialogEnabled.rawValue: false,
+      OptimizelyFeature.facebookConversionsAPI.rawValue: false,
```

Comment:
Yes we can remove this since the CAPI code is gated behind the user's consent preferences instead of this flag